packages/components/src/dateInput/DateInput.js

```diff
@@ -59,7 +59,9 @@ export default class DateInput extends React.Component<Props, State> {
     return (
       <View>
         {label != null && (
-          <FormLabel style={labelContainerStyle}>{label}</FormLabel>
+          <FormLabel labelContainerStyle={labelContainerStyle}>
```

Comment:
Maybe you could make the new prop available in these other components too, even though that's not the scope of my request. @jvaclavik WDYT?

Reply:
Sounds good!